packages/ui/src/ui/pages/navigation/modals/RemoteCopyModal.tsx

```diff
@@ -186,6 +185,23 @@ function RemoteCopyModal() {
                     name: 'pool',
                     type: 'pool',
                     caption: 'Pool',
+                    warning: (
+                        <>
+                            pool must have specified{' '}
+                            {docsUrl(
+                                makeLink(
+                                    UIFactory.docsUrls['operations:remote_copy'],
+                                    'limits',
+                                    true,
+                                ),
+                            )}
+                        </>
+                    ),
+                    visibilityCondition: {
+                        when: 'dstCluster',
+                        isActive: (value) => Boolean(value),
+                    },
+                    required: true,
```

Comment:
According to the documentation
https://ytsaurus.tech/docs/en/user-guide/data-processing/operations/operations-options#common_options
the parameter might be omitted.